imap_processing/quality_flags.py

```diff
@@ -0,0 +1,63 @@
+"""Bitwise flagging."""
+
+from enum import IntFlag
+
+
+class L1bQualityFlags(IntFlag):
```

Comment:
I will look at this first thing on Monday. Can you request input from Tim as well since he may have good idea on this topic?